my/lessons/basics/enum.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: page
+title: Enum
+category: basics
+order: 3
+lang: en
```

Comment:
For the time being can you remove these `en` lessons or change the lang to `my`?  Unfortunately this way will cause issues with the site.